teku/src/main/java/tech/pegasys/teku/cli/options/BeaconRestApiOptions.java

```diff
@@ -112,15 +112,13 @@ public void setMaxUrlLength(int maxUrlLength) {
     this.maxUrlLength = maxUrlLength;
   }
 
-  // beacon-liveness-tracking-enabled
   @Option(
-      names = {"--Xbeacon-liveness-tracking-enabled"},
+      names = {"--beacon-liveness-tracking-enabled"},
```

Comment:
lets make this able to accept both and avoid some of the dramas from block-timing-tracking

Reply:
Done. But that means that the `--Xbeacon-liveness-tracking-enabled` will be printed in the help menu too (since it's not hidden anymore)

Reply:
we can still document that its deprecated and will be removed later, just hoping that it reduces confusion - not 100% sold either way, whether we rip the bandaid off and answer all the questions, or whether better off hoping not too many people are currently using it...

Reply:
we should probably have a team discussion around how we want to handle these generally, as a lot of people got caught out by the block timings one...